docker_boot.sh

```diff
@@ -1,5 +1,14 @@
 #!/usr/bin/env bash
 
-./wait-for-it.sh database2:3306 -- php artisan migrate
+set -e
 
-php-fpm
+DB_HOST="${DB_HOST:?environment variable is missing}"
+DB_PORT="${DB_PORT:-3306}"
+
+./wait-for-it.sh "${DB_HOST}:${DB_PORT}" -- php artisan migrate --force
+
+# php artisan optimize
+
+php artisan storage:link
+
+exec $@
```

Comment:
What is this supposed to do?

Reply:
This will replace the current process with whatever is passed to the entrypoint.
So the Dockerfile sets the entrypoint to this docker_boot.sh script and the cmd is the actual supervisord in blocking mode.

This just mimics what is done in the upstream package and their docker-entrypoint.sh. If we have no use for overriding the CMD we could hardcode the supervisord command here